recipes/fontconfig/meson/conanfile.py

```diff
@@ -75,15 +74,14 @@ def generate(self):
             "doc": "disabled",
             "nls": "disabled",
             "tests": "disabled",
-            "tools": "disabled"
+            "tools": "disabled",
+            "sysconfdir": os.path.join(self.package_folder, "res", "etc"),
+            "datadir": os.path.join(self.package_folder, "res", "share"),
```

Comment:
```suggestion
            "sysconfdir": os.path.join("res", "etc"),
            "datadir": os.path.join("res", "share"),
```
according to https://c3i.jfrog.io/c3i/misc/logs/pr/23010/8-linux-gcc-legacy/fontconfig/2.14.2//1dd5f947341f0c35b104c3cbc22073e69527248e-build.txt
these two paths seem to be relative
see also https://gitlab.freedesktop.org/fontconfig/fontconfig/-/blob/2.14.2/meson.build?ref_type=tags#L291 and 
https://gitlab.freedesktop.org/fontconfig/fontconfig/-/blob/2.14.2/meson.build?ref_type=tags#L301

Reply:
Oh yeah, I'll need to update the version requirement too.